types/node/http.d.ts

```diff
@@ -1404,14 +1404,14 @@ declare module "http" {
          * The 3-digit HTTP response status code. E.G. `404`.
          * @since v0.1.1
          */
-        statusCode?: number | undefined;
+        statusCode?: number | undefined | null;
         /**
          * **Only valid for response obtained from {@link ClientRequest}.**
          *
          * The HTTP response status message (reason phrase). E.G. `OK` or `Internal Server Error`.
          * @since v0.11.10
          */
-        statusMessage?: string | undefined;
+        statusMessage?: string | undefined | null;
```

Comment:
Since these properties are set to `null` in the constructor, can these ever actually be absent? Would this be more accurate as eg. `statusCode: number | null`?

Reply:
I think number | null is better.